force-app/main/default/classes/OpportunityUtils_Test.cls

```diff
@@ -7,4 +7,110 @@ private class OpportunityUtils_Test
 
     // 100% coverage can be achieved without testing all scenarios,
     // so don't stop just because you have full coverage.
+
+
+    //Tests all scenarios for a variety of amounts in 200 records
+    @isTest
+    private static void testBulk()
+    {
+        User salesUser = UnitTestHelper.CreateUser_inMemory('Joe', 'Bloggs', 'jbl', 'Standard User');
+        insert salesUser;
+
+        Account testAcc = UnitTestHelper.createAccount_InMemory();
+        insert testAcc;
+
+        //Creates 200 opps with amounts 1k to 200k
+        List<Opportunity> testOpps = UnitTestHelper.createOpps_InMemory(testAcc, 200);
+
+        Test.StartTest();
+        System.runAs(salesUser)
+        {
+            OpportunityUtils oppUtils = new OpportunityUtils();
+            oppUtils.addOpportunitySizeToNameAndSave(testOpps); 
+            //Why isn't this a static method?
```

Comment:
In this case there's no particular reason why you have to instantiate an object and then call static methods. As a general principle they are more extensible, as you could inherit from the OpportunityUtils class and override the method for specific use cases. I would typically put this behind an interface so that I could mock it out for unit tests and reduce coupling (which we'll cover this week) but I decided in this case it would add complexity for no benefit.